content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst

```diff
@@ -0,0 +1,146 @@
+corner of the mailing list detail form.
+
+.. image:: mailing_lists_blacklists/sms-mailing-list.png
+   :align: center
+   :alt: View of the mailing list template form in Odoo SMS Marketing.
+
+.. note::
+   Don't forget to hit the :guilabel:`Save` button once changes have been made.
+
+Mailing lists contacts
+======================
+
+If :guilabel:`Mailing List Contacts` is selected from the :guilabel:`Mailing Lists` header drop-down
+menu, Odoo reveals a separate page with all the contacts associated with one (or more) of the
+configured mailing lists in the database.
+
+.. image:: mailing_lists_blacklists/mailing-list-contacts-page.png
+   :align: center
+   :alt: View of the mailing lists contact page in the Odoo SMS Marketing application.
+
+.. note::
+   By default, Odoo reveals the :guilabel:`Mailing List Contacts` page with the :guilabel:`Exclude
+   Blacklisted Phone` filter in the search bar. Therefore only showing contact information for
+   recipients who still want to receive communications and mailings.
+
+Communication history in the Chatter
+------------------------------------
+
+An accessible record of the sent mailings is kept on each recipient's :guilabel:`Chatter`, located
+beneath a recipient's contact form (in the :guilabel:`Contacts` application).
+
+Database users can reference the :guilabel:`Chatter` to easily keep track of communications, and see
+a history of interactions with contacts and prospects.
+
+For example, sales representatives can use the :guilabel:`Chatter` to quickly find out which SMS
+mailing promotions a certain customer has received (or not received).
+
+.. image:: mailing_lists_blacklists/sms-marketing-chatter.png
+   :align: center
+   :alt: View of what the chatter looks like in the Odoo Contacts app.
+
+Blacklist
+=========
+
+:guilabel:`Odoo SMS Marketing` has a :guilabel:`Blacklist` feature that provides recipients with the
+power to add their phone number to a list of people who no longer want to receieve communications or
+mailings.
+
+This is also known as the "Unsubscribe Process." Customers will automatically be added onto the
+:guilabel:`Blacklist`, if they click :guilabel:`Unsubscribe`, via their :guilabel:`Subscription
+Management Page`. Customers can also be added manually to the blacklist, if necessary.
+
+To see a complete collection of blacklisted numbers, navigate to the :guilabel:`SMS Marketing` app,
+and go to :menuselection:`Configuration --> Blacklisted Phone Numbers`. Doing so, reveals a separate
+page containing every blacklisted phone number in the database.
+
+.. image:: mailing_lists_blacklists/sms-blacklist-menu.png
+   :align: center
+   :alt: SMS Blacklist menu in the application.
+
+To manually add a number, click the :guilabel:`Create` button in the upper-left. When clicked, Odoo
+reveals a separate page, in which the phone number to be blacklisted is entered. There's also a
+checkbox to indicate whether that particular phone numnber is :guilabel:`Active` (or not).
```

Comment:
"Odoo reveals a separate page" phrase is too wordy. Just say what's happening in clear terms 😉 
```suggestion
To manually add a number to a blacklist, click the :guilabel:`Create` button in the upper-left
corner of the dashboard and enter the phone number on the next page's form. There's also a
```